components/admin/orgs/OrganisationControlForm.tsx

```diff
@@ -1,93 +1,77 @@
 import { Box, useToast, useDisclosure } from '@chakra-ui/react'
-import {
-  getFromUrlArrayAndParseJson,
-  getFromUrlStringAndParseJson,
-  isUserLoggedIn,
-  makeFetchToUrlWithAuth,
-} from '../../../utils'
+import { makeFetchToUrlWithAuth } from '../../../utils'
 import { useUserInfo } from '../../../hooks/useUserInfo'
-import useSWR from 'swr'
-import useSWRImmutable from 'swr/immutable'
+import { KeyedMutator } from 'swr'
 import { useState } from 'react'
 import OrganisationControlFormPopup from './OrganisationControlFormPopup'
 import defaultValues from './initialValues'
 import validationSchema from './validationSchema'
-import { makeSuccessOrgToast, makeErrorOrgToast } from '../../../utils/orgUtils'
+import { makeSuccessToast, makeErrorToast } from '../../../utils/orgUtils'
 import AdminTable, { AdminTableColumnProps } from '../AdminTable'
 
-function OrganisationControlForm() {
+type OrganisationControlFormProps = {
+  users: any[]
+  orgs: OrganisationWithIGHead[]
+  categories: any
```

Comment:
To fix this

Reply:
Fixed and tested in 867b13d723b48bca01ba2b189aaaa51285bfb597